reactome_analysis_datasets/reactome_analysis_datasets/dataset_fetchers/expression_atlas_fetcher.py

```diff
@@ -30,6 +31,9 @@
 import os
 import time
 import signal
+import requests
+
+from reactome_analysis_api.models.external_data_sample_metadata import ExternalDataSampleMetadata
```

Comment:
Ideally, imports should be in some kind of order (just for the future)
ie. put all internal imports at the top and system imports afterwards or vice versa

Reply:
Btw. I don't think that we need this import at all.

Reply:
True not needed anymore